src/tests/Search.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { Search } from '../components/Search.tsx'
+import { render, screen } from '@testing-library/react'
+import { userEvent } from '@testing-library/user-event'
+
+describe('Search', () => {
+    const user = userEvent.setup()
```

Comment:
It is discouraged to call `setup` method outside of a test. Read more about it here: https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent